Apsitvarkom.Api/package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I am uncertain and curious whether we do really need this file. Maybe @BenasB could contribute on this matter?

Reply:
Accidently added. Removed on next commit